scss/detail.scss

```diff
@@ -56,3 +57,22 @@
 .book-separator {
   border-bottom: 4px solid $earls-green;
 }
+
+.book-image-container {
+  &::after {
+  -ms-transform: rotate(15deg);
+  -webkit-transform: rotate(15deg);
+  background: url('../assets/badge.png');
+  background-repeat: no-repeat;
+  background-position: 1px;
+  background-size: 100%;
```

Comment:
Just a comment here to take into account. You can use a short hand for this

`background: url('../assets/b nadge.png') no-repeat center/cover;`